configargparse.py

```diff
@@ -119,6 +121,44 @@ def serialize(self, items):
         """
         raise NotImplementedError("serialize(..) not implemented")
 
+    def _tweak_value(self, key, value, filename):
```

Comment:
Having two two methods with the same name except for the _ prefix is not something I've seen before. Unless this is some naming convention I'm not aware of, I'd vote for more a more explicit name difference (eg. these are not the best suggestions, but perhaps _tweak_value_base or alternatively .tweak_value_impl)

Reply:
This is used for `parse_known_args`, `_parse_known_args2` and `_parse_known_args` in the [underlying class](https://github.com/python/cpython/blob/main/Lib/argparse.py) but I'd be happy to rename this pseudo-private function to something more explicit.